### DISCUSSION TITLE
Build done!

### DISCUSSION THREAD

**Discussion by gabhalling** [rdudhagra/Sand-Table#56] [Q&A] [answered]
Finally got done building the thing! It was a really fun time!

The issues I have right now are with file uploading. I wanted to be able to upload both .thr and .gcode so I used some modified parts in the fork iwannabswiss made. But now I can only upload .gcode files... When I try to upload a .thr file, the website crashes... If anyone got the same issue, let me know.

![image](https://github.com/user-attachments/assets/e1c499a3-47cc-45c5-9794-a7526f6d90ee)

Here are some pictures of the finished table: 

![20240901_120143](https://github.com/user-attachments/assets/b7da5f5b-03f9-4d02-81d8-83e3aaf8a6a8)
![20240831_162852](https://github.com/user-attachments/assets/2ff846d3-0cf3-4dc1-bf01-8da9dd43b25c)
![20240901_073845](https://github.com/user-attachments/assets/f820deac-cea7-4ca9-8d1d-21904f147c3d)

Thank you Ravi for such an awesome project!

**Comment by rdudhagra** [comment, marked as answer]
Looks amazing Gab! Glad it all came together. As far as the website error, I think I found the bug in the fork you're using. Let me know if [this](https://github.com/rdudhagra/Sand-Table/discussions/25#discussioncomment-10524204) turns out to be the issue.

**Reply by gabhalling** [reply]
Thanks! I'll try it when I get the time! For now, I ended up converting the files from .thr to .gcode through Sandify and that seems to work for me. 

Only thing I wish would be possible now is for Sisyphus to make their Community Tracks/App public or not require a whole table that they make... Sandify works nicely for symmetrical designs but for drawings and such I have yet to find a good software to free hand draw stuff over a template or something... 

Anyway, here's 2 drawings that I managed to make by myself so far:

![20240902_133342](https://github.com/user-attachments/assets/791fc161-7e16-4616-a6b0-11f6c7c909b9)
![20240902_142617](https://github.com/user-attachments/assets/e29bad4f-be5f-4f66-ac48-49092338698e)

**Comment by mattYogi** [comment]
I found that some of the larger thr files needed more memory during conversion. My 2GB Raspberry Pi crashed when I tried to convert the larger files. Switching to a 4 GB Pi seemed to have solved the issue. 

Your table looks great.

**Reply by rdudhagra** [reply]
Nice, thanks for the note. Bumping up the swap size on the Pi will also help with this (the default of 100M is kinda small). Although this will wreck your SD card after a while.
